plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1015,7 +987,7 @@ int amdgpu_plugin_dump_file(int fd, int id)
 			goto failed;
 		}
 
-		thread_datas[i].drm_fd = open_drm_render_device(dev->drm_render_minor);
+		thread_datas[i].drm_fd = node_get_drm_render_device(dev->drm_render_minor);
```

Comment:
can you rename it to thread_args or something else, datas doesn't sound very nice.